schema/NPSSchema.mcf

```diff
@@ -0,0 +1,187 @@
+descriptionUrl: “https://www.bjs.gov/index.cfm?ty=tdtp&tid=1”
+
+Node: dcid:NPSCauseOfDeathEnum
+name: "NPSCauseOfDeathEnum"
+typeOf: schema:Class
+subClassOf: schema:Enumeration
+description: "Enumeration for NPS measured causes of death"
+
+Node: dcid:JudicialExecution
+name: "JudicialExecution"
+typeOf: dcs:NPSCauseOfDeathEnum
+description: "Execution for committing a crime"
+
+Node: dcid:IllnessOrNaturalCause
+name: "IllnessOrNaturalCause"
+typeOf: dcs:NPSCauseOfDeathEnum
+description: "Illness or other natural cause of death excluding AIDS"
+
+Node: dcid:AIDS
+name: "AIDS"
+typeOf: dcs:NPSCauseOfDeathEnum
+description: "AIDS"
+
+Node: dcid:Suicide
+name: "Suicide"
+typeOf: dcs:NPSCauseOfDeathEnum
+description: "Suicide"
+
+Node: dcid:AccidentalInjuryToSelf
+name: "AccidentalInjuryToSelf"
+typeOf: dcs:NPSCauseOfDeathEnum
+description: "Accidentally caused injury to oneself"
+
+Node: dcid:AnotherPerson
+name: "AnotherPerson"
+typeOf: dcs:NPSCauseOfDeathEnum
+dsescription: "Another person caused the death"
+
+Node: dcid:Homicide
+name: "Homicide"
+typeOf: NPSCauseOfDeathEnum
+description: "Homicide"
+
+Node: dcid:NPSOtherCauseOfDeath
+name: "NPSOtherCauseOfDeath"
+typeOf: dcs:NPSCauseOfDeathEnum
+description: "Other cause of death not already listed"
+
+Node: dcid:correctionalFacilityOperator
+name: "CorrectionalFacilityOperator"
+typeOf: schema:Property
+rangeIncludes: dcs:CorrectionalFacilityOperatorEnum
+domainIncludes: dcs:StatisticalVariable
+description: "The level that the correctional facility is operated at"
+
+Node: dcid:CorrectionalFacilityOperatorEnum
+name: "CorrectionalFacilityOperatorEnum"
+typeOf: schema:Class
+subClassOf: schema:Enumeration
+description: "Enumeration of the different levels correctional facilities are operated at"
+
+Node: dcid:PrivatelyOperated
+name: "PrivatelyOperated"
+typeOf: dcs:CorrectionalFacilityOperatorEnum
+description: "Operate at the private level"
+
+Node: dcid:LocallyOperated
+name: "LocallyOperated"
+typeOf: dcs:CorrectionalFacilityOperatorEnum
+description: "Operate at the local level"
+
+Node: dcid:StateOperated
+name: "StateOperated"
+typeOf: dcs:CorrectionalFacilityOperatorEnum
+description: "Operate at the state level"
+
+Node: dcid:FederallyOperated
+name: "FederallyOperated"
+typeOf: dcs:CorrectionalFacilityOperatorEnum
+description: "Operate at the federal level"
+
+Node: dcid:CorrectionalFacilityLocation
+name: "CorrectionalFacilityLocation"
+typeOf: schema:Property
+rangeIncludes: dcs:CorrectionalFacilityLocationEnum
+domainIncludes: dcs:StatisticalVariable
+description: "Where the correctional facility is located relative to a location"
+
+Node: dcid:CorrectionalFacilityLocationEnum
+name: "CorrectionalFacilityLocationEnum"
+typeOf: schema:Class
+subClassOf: schema:Enumeration
+description: "Enumeration of relative locations of correctional facilities"
+
+Node: dcid:InState
```

Comment:
These seem too general, might consider making them specific to correctional facility, as:
InStateCorrectionalFacility, LocalCorrectionalFacility, etc.

Reply:
Ditto for LocallyOperated, FederallyOperated, etc. above.